.travis/check-posix-app-pty

```diff
@@ -67,37 +70,48 @@ check() {
 
     RADIO_NCP_PATH="$(pwd)/$(ls output/*linux*/bin/ot-ncp-radio)"
     $RADIO_NCP_PATH 1 > $RADIO_PTY < $RADIO_PTY &
-    OT_NCP_PATH="$(pwd)/$(ls output/posix/*linux*/bin/ot-ncp)"
-    PLATFORM_NETIF=wpan0 sudo -E wpantund -I wpan0 -o Thread:Config:FilterRLOCAddresses 0 -s "system:${OT_NCP_PATH} ${CORE_PTY}" &
-
+    OT_CLI_PATH="$(pwd)/$(ls output/posix/*linux*/bin/ot-cli)"
+    sudo expect <<EOF > $OT_OUTPUT &
+spawn $OT_CLI_PATH ${OT_NCP_PATH} ${CORE_PTY}
```

Comment:
nit: align?

Reply:
This is *here document*, if indent, it will result in a lot of in leading spaces, and the final `EOF` cannot indent.